receipt_agent/receipt_agent/subagents/metadata_finder/tools/receipt_metadata_finder.py

```
@@ -935,6 +966,182 @@ async def apply_fixes(
 
         return result
 
+    async def _create_receipt_place_from_match(
+        self, match: MetadataMatch, metadata: "ReceiptMetadata"
```

Ruff (0.14.8)

Undefined name `ReceiptMetadata`

(F821)